.github/workflows/e2e.yml

```diff
@@ -280,7 +280,7 @@ jobs:
         env:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
           SLACK_COLOR: 'fail'
-          SLACK_MESSAGE: ${{ matrix.project }} failed on ${{ github.head_ref || github.ref_name }} https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}/jobs/${{ github.job }}
+          SLACK_MESSAGE: 'E2E job `${{ matrix.project }}` failed on `${{ github.head_ref || github.ref_name }}` branch, details: https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
```

Comment:
The variable `${{ github.job }}` did not work as expected, so I'm switching the link to point to the summary page, which also shows the artifacts for the errors. Also adding more description for the message.